src/modules/main/guilds/_guilds.scss

```diff
@@ -88,6 +88,22 @@
             }
         }
 
+        // Inbox
+        .iconButton-2z1iK7 { // NOTE 2022-03-07 - Updated Inbox icon when Experiment 2022-01_inbox_redesign is on. No hover effect however unlike the Home button. Please check.
+            &[aria-label="Inbox"] {
+                background-color: transparent;
+                &::before{
+                    content: "\E715"; // Mail
+                    color: $TextFillColorPrimary;
+                    @include FontIconFluent;
+                    font-size: 16px;
+                }
+                svg{
+                    display: none;
+                }
+            }
+        }
+
```

Comment:
Can remove this section - accidentally resolved in af043b8466f3952bf5366c42d80a9580b607f9b8